code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -13,18 +16,43 @@
 			if(welded)
 				overlays += "welded"
 		else
+			if(exploded)
+				icon_state = icon_exploded
+				return
 			icon_state = icon_opened
 
+//Fridges cannot be destroyed by explosions (a reference to Indiana Jones if you don't know)
+//However, the door will be blown off its hinges, permanently breaking the fridge
+//And of course, if the bomb is IN the fridge, you're fucked
 /obj/structure/closet/secure_closet/freezer/ex_act(var/severity)
-	// IF INDIANA JONES CAN DO IT SO CAN YOU
 
-	// Bomb in here? (using same search as space transits searching for nuke disk)
+	//Bomb in here? (using same search as space transits searching for nuke disk)
 	var/list/bombs = search_contents_for(/obj/item/device/transfer_valve)
 	if(!isemptylist(bombs)) // You're fucked.
 		..(severity)
 
+	if(severity == 1)
```

Comment:
I can't believe it's not defines.
This won't block your PR, but it's still nonsense.